lib/src/sdk/dart.dart

```diff
@@ -31,6 +39,21 @@ class DartSdk extends Sdk {
     return aboveExecutable;
   }();
 
+  /// The loaded `sdk_packages.yaml` file if present.
+  static final SdkPackageConfig? _sdkPackages = () {
+    var path = p.join(_rootDirectory, 'sdk_packages.yaml');
```

Comment:
Should we not make this a `.json` file? Are we getting any real advantage from yaml here.

Reply:
I don't particularly care, I do expect it to be hand written and yaml is more readable/editable. But happy to go with whatever you all prefer, cc @jonasfj .

Reply:
If pub is the only consumer, I won't care.
The `package_config.json` file is JSON because every source processing til needs to read it.